galaxy_importer/loaders.py

```diff
@@ -134,54 +201,45 @@ def handle_nested_tables(obj, table_key):
                 for row in obj[table_key]:
                     handle_nested_tables(row, table_key)
 
-        doc = data[self.name].get('doc')
+        doc = data.get('doc', None)
```

Comment:
Suppose you could make the default return an empty dict  `doc = data.get('doc', {})`.  Would eliminate the 'if doc' condition. And `'options' in doc.keys()` could be `'options' in doc`

But if it works, I'd likely live it alone for now. If the output format of ansible-doc is changing over time, it is hard to avoid having lots of conditional checks. And if those checks are working, don't touch em (unless there are good tests for both output formats).